loops-arrays-objects.js

```diff
@@ -82,17 +82,18 @@ for (let i = 0; i < scores.length; i++) {
 
```

Comment:
Ik kan hierboven niet commenten, maar bij 1b, als ik het test voor een score van 100 dan werkt hij niet. Ik zou daar een else statement van maken, dus zonder condition. Je hebt immers de rest al beschreven. Geldt ook voor 1c.
En het lijkt alsof je een { vergeten bent bij de laatste if. Hij werkt wel, maar is minder netjes :)